minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -280,6 +282,10 @@ public void stop() throws Exception {
 
     resetContext();
     resetLoginUser();
+    if (mHome != null) {
```

Comment:
Yes, but JVM crashes are quite rare, and this is all happening inside a "temporary" directory anyway, so I think we should be fine. If it becomes a problem we can always attempt cleanup on startup.